charts/graylog/values.yaml

```diff
@@ -213,6 +213,26 @@ graylog:
       ##
       type: ClusterIP
 
+    startupProbe:
```

Comment:
Wrong indent. It cause a build fail
https://github.com/KongZ/charts/runs/5283360109?check_suite_focus=true

Reply:
Thanks! My bad. Should be fixed now.